puck.config.tsx

```diff
@@ -427,10 +444,30 @@ export const config: Config<Props, RootProps> = {
       fields: {
         strapline: { type: "textarea" },
         description: { type: "textarea" },
+        headingLevel: {
+          type: "select",
+          options: [
+            { label: "", value: "" },
+            { label: "H1", value: "1" },
+            { label: "H2", value: "2" },
+            { label: "H3", value: "3" },
+            { label: "H4", value: "4" },
+            { label: "H5", value: "5" },
+            { label: "H6", value: "6" },
+          ],
+        },
       },
-      render: ({ description = "description", strapline = "strapline" }) => (
+      render: ({
+        description = "description",
+        headingLevel,
+        strapline = "strapline",
+      }) => (
         <Section>
-          <Hero description={description} strapline={strapline} />
+          <Hero
+            description={description}
+            headingLevel={headingLevel}
+            strapline={strapline}
```

Comment:
Thoughts on dropping the Hero lock-up in favour of compositional Header and Paragraph, which now have max width support?

I was also going to suggest we keep the CTA on desktop - I don't know if it matters if there's a dupe.

Reply:
I suggest let's wait for Dropzones. But I do think having a Hero component is not a bad thing, as you often want to tweak the Hero behaviour beyond what the underlying typography can do, especially responsively, just cause it's the Hero and therefore very visible and important. Also I'm looking at Heading and Paragrpah next, and may want to remove or tweak the `maxWidth` props there anyway possibly replace with an `isMeasured` prop! Agree there's potential to drop it, but suggest give it a bit of time. On the CTA I don't think we need to double it on desktop, but if we did, I think the styling would need to change to reduce the double button effect, and also make it more propotional with the strapline. But I don't think we really need it and cleaner without.

Reply:
I like `isMeasured`!

Sounds good re CTA.